sql/catalyst/src/test/scala/org/apache/spark/sql/types/DataTypeTestUtils.scala

```diff
@@ -46,6 +46,25 @@ object DataTypeTestUtils {
    */
   val numericTypes: Set[NumericType] = integralType ++ fractionalTypes
 
+  // TODO: remove this once we find out how to handle decimal properly in property check
+  val numericTypeWithoutDecimal: Set[DataType] = integralType ++ Set(DoubleType, FloatType)
+
+  /**
+   * Instances of all [[NumericType]]s and CalendarIntervalType
```

Comment:
put [[ ]] around CalendarIntervalType so IntelliJ can find it during refactoring